tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+import shlex
+import sqlalchemy
+
+from unittest.mock import patch
+
+from pymdmix_core.parser import get_mdmix_parser
+from pymdmix_core.plugin.base import PluginManager
+
+your_plugin_name='project'
```

Comment:
this should point to 'plugin_template' instead of project, it won't pass the tests otherwise